comp/otelcol/converter/impl/testdata/connectors/dd-connector/config-result.yaml

```diff
@@ -0,0 +1,58 @@
+receivers:
```

Comment:
can we move this file in the `processors` folder ? Technically we are still testing processor, even though bug has to do with connector